tutorial/coupled_pendulum/README.md

```diff
@@ -0,0 +1,38 @@
+# Run coupled pendulum tutorial in Gazebo
+
```

Comment:
Somewhere here we should document that this model has a runtime dependency on `icub-main`.